File: Assignments September 2024/Prog88.rb
puts "Insert number 1-13"
a = gets.chomp.to_i
puts "Insert number 13-20"
b = gets.chomp.to_i
puts "Origin numbers are: " + a.to_s + " and " + b.to_s
sum = a + b
difference = a - b
product = a * b
average = sum / 2
absolute = (a - b)
if absolute < 0
    absolute = absolute * -1
else 
    absolute = absolute * 1
end
if a > b
    max = a
    min = b
else
    max = b
    min = a
end
puts "The sum is:" + sum.to_s
puts "The difference is:"+ difference.to_s
puts "The product is:"+ product.to_s
puts "The average is:"+ average.to_s
puts "The absolute is:"+ absolute.to_s
puts "The max is:"+ max.to_s
puts "The min is:"+ min.to_s

File: Assignments September 2024/Prog88.py
#Mini calculator
def variables():
    a = int(input("Input number A: "))
    b = int(input("Input number B: "))
    minicalc(a,b)
    
def minicalc(a, b):
    sum = a + b
    diff = a - b
    pro = a * b
    ave = sum / 2
    dis = abs(diff)

    if a > b:
        max = a
        min = b
    else:
        max = b
        min = a
    answer(sum, diff, pro, ave, dis, max, min)

def answer(sum, diff, pro, ave, dis, max, min):
    print("The sum is: " + str(sum))
    print("The difference is: " + str(diff))
    print("The product is: " + str(pro))
    print("The average is: " + str(ave))
    print("The distance is: " + str(dis))
    print("The maximum is: " + str(max))
    print("The minimum is: " + str(min))

c = input("Do you want to start calc? y/n ")
if c == "y":
    variables()
else:
    print("Fine then. Be that way.")